src/client/components/Header.js

```diff
@@ -1,11 +1,8 @@
 import React from 'react';
 
 import AppIcon from '../assets/images/coffee-solid.svg';
-import {
-  COMPANY_NAME,
-  COMPANY_PAGE_URL,
-  PROJECT_NAME
-} from '../common/variables';
+import { COMPANY_PAGE_URL, PROJECT_NAME } from '../common/variables';
+import SandstreamLogo from '../assets/images/sandstream_logo.png';
```

Comment:
please change to something more general like CompanyLogo and also change file name